Fix RPC Cors (#3520)
freqtrade/freqtrade: docs/rest-api.md

> curl -X POST --header "Authorization: Bearer ${refresh_token}"http://localhost:8080/api/v1/token/refresh
{"access_token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE1ODkxMTk5NzQsIm5iZiI6MTU4OTExOTk3NCwianRpIjoiMDBjNTlhMWUtMjBmYS00ZTk0LTliZjAtNWQwNTg2MTdiZDIyIiwiZXhwIjoxNTg5MTIwODc0LCJpZGVudGl0eSI6eyJ1IjoiRnJlcXRyYWRlciJ9LCJmcmVzaCI6ZmFsc2UsInR5cGUiOiJhY2Nlc3MifQ.1seHlII3WprjjclY6DpRhen0rqdF4j6jbvxIhUFaSbs"}
```

## CORS

All web-based frontends are subject to [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) - Cross-Origin Resource Sharing.

> Since most request to the Freqtrade API must be authenticated, a proper CORS policy is key to avoid security problems.

Reviewer: ```suggestion
Since most of the requests to the Freqtrade API must be authenticated, a proper CORS policy is key to avoid security problems.
```
(or "most requests")

> Also, the Standard disallows `*` CORS policies for requests with credentials, so this setting must be done appropriately.

Reviewer: ```suggestion
Also, the standard disallows `*` CORS policies for requests with credentials, so this setting must be set appropriately.
```

> Users can configure this themselfs via the `CORS_origins` configuration setting.

Reviewer: ```suggestion
Users can configure this themselves via the `CORS_origins` configuration setting.
```

It consists of a list of allowed sites that are allowed to consume resources from the bot's API.

> Assuming your Application would be deployed as `https://frequi.freqtrade.io/home/` - this would mean that the following configuration becomes necessary:

Reviewer: ```suggestion
Assuming your application is deployed as `https://frequi.freqtrade.io/home/` - this would mean that the following configuration becomes necessary:
```
